src/styles/index.scss

```diff
@@ -3,3 +3,4 @@
 @import url('./navbar.scss');
 @import url('./cards.scss');
 @import url('./company.scss');
+@import url('./search.scss')
```

Comment:
As required, lets stick to the design template given 👍 

> Your design

![image](https://user-images.githubusercontent.com/66112760/148460268-b46674cd-a24a-4814-b13d-5c1eab58c432.png)

> Original design template 
![image](https://user-images.githubusercontent.com/66112760/148460470-6215dde1-f806-43e3-9811-fc5aa624117f.png)

Kindly follow these design guidelines, including:
 - Colors (select one main color for your website).
 - Typography: font face, size and weight.
 - Layout: composition and space between elements.